src/bp-groups/bp-groups-filters.php

```diff
@@ -100,6 +100,7 @@
 // Filter group count.
 add_filter( 'bp_groups_get_where_count_conditions', 'bb_groups_count_update_where_sql', 10, 2 );
 
+add_action( 'bp_after_group_activity_content', 'bb_group_activity_pinpost_confirmation_modal_template' );
```

Comment:
remove this filter from here and move this to https://github.com/buddyboss/buddyboss-platform/pull/4156/files#diff-694325250d760edf42e4c7c1d3279cd75143a3227f8e547af9e27c940133f18cR176

Also, remove the call back for the group activity specific.